content/terraform/v1.12.x/docs/language/values/variables.mdx

```diff
@@ -76,7 +68,9 @@ variable "environment" {
 
 </CodeBlockConfig>
 
-The `environment` variable contains an optional `validation` block to ensure the value a consumer assigns meets module requirements. Learn more about [validating variable values in your configuration](/terraform/language/validate).
+The `environment` variable also contains an optional `validation` block to ensure the value a consumer inputs meets module requirements. Learn more about [validating variable values in your configuration](/terraform/language/validate).
```

Comment:
```suggestion
The `environment` variable also contains an optional `validation` block to ensure the value a consumer specifies as the input value meets module requirements. Learn more about [validating variable values in your configuration](/terraform/language/validate).
```
I made this comment earlier, but "input" as a noun and verb is easy to muddle. To me, the "input" as an adjective is less confusing.